### DISCUSSION TITLE
I created a python wrapper for the KAGI API.

### DISCUSSION THREAD

**Discussion by WooilJeong** [kagisearch/kagi-docs#50] [Q&A]
Hello, I saw the official Kagi API documentation and thought it would be nice to have a Python client, so I wrote Python wrapper code for the Universal Summarizer API and Search API. You can find the code on GitHub and PyPI. I'd love to contribute to the Kagi API project and would appreciate any suggestions on how to do so. Thank you. 😀

- PyKagi GitHub: https://github.com/WooilJeong/PyKagi
- PyKagi PyPI: https://pypi.org/project/PyKagi/

**Comment by z64** [comment]
Hi @WooilJeong, thanks for submitting!

I think this might be a good time for us to start a list of open-source packages in our docs that users can pick up.
